cerberus/tests/test_bookings.py

```diff
@@ -155,21 +134,22 @@ def test_pet_count(walk_service, make_pet):
         end=BookingSlot.round_date_time(datetime.now() + timedelta(hours=6)),
         booking_slot=None,
         service=walk_service,
+        pets=[make_pet()],
     ).booking_slot
 
     assert booking_slot_1.pk == booking_slot_2.pk
     assert booking_slot_1.pet_count == 2
 
 
 @pytest.mark.django_db
-def test_customer_count(walk_service, make_pet, customer):
+def test_customer_count(walk_service):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Consider adding more context or a docstring to `test_customer_count`.

It's not immediately clear what specific behavior or edge case this test is verifying. Adding a brief explanation could improve readability.